### DISCUSSION TITLE
Email Address for Arnold

### DISCUSSION THREAD

**Discussion by sm01-uk** [Arnold-n/P1P2MQTT#132] [General]
Hi Arnold - I emailed you at what I thought was your email address but haven't had a reply - interested in P1P2 MQTT bridge and would like to know more about buying the hardware from you. Would you be able to get back to me with an email address which I can reach you on?

Thanks,
Steve.

**Comment by Arnold-n** [comment]
Hi Steve, sorry, your mail slipped through, I just responded to you.
